reflex/components/next/image.py

```diff
@@ -80,6 +83,27 @@ def create(
         Returns:
             _type_: _description_
         """
+        if "width" not in props or "height" not in props:
+            # props.pop("width", None)
+            # props.pop("height", None)
+            if "width" in props:
+                props["width"] = f"{props['width']}px"
```

Comment:
We should only do this if they're not already setting a unit

Reply:
next_image only accept int anyway so they can't set unit